src/components/HiddenText/HiddenText.module.scss

```diff
@@ -1,8 +1,8 @@
 @import "../../styles/themes";
 @import "../../styles/typography";
 
-.hidden-text--wrapper {
-  // clip pattern taken from https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/.
+.wrapper {
+  /* clip pattern taken from https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/. */
```

Comment:
why CSS comment and not SCSS comment?

Reply:
Script is changing all comments to this pattern due to errors which occurred after script execution - not correct line breaks and smth else

Reply:
i recommend correcting this.